src/main/java/seedu/bigpp/parser/Parser.java

```diff
@@ -57,11 +59,15 @@ private Command parseMainMenuCommand(String userInput) {
 
     private Command parseViewerCommand(String userInput) {
         String commandWord = userInput.split(" ")[0];
+        String arguments = userInput.split(" ")[1];
```

Comment:
Can try putting this line below instead in the case: "add" as other commands such as "bye"(to be implemented in the future does not have an argument. Should handle exceptions for this case.

Reply:
#34 Create issue and will solve in future iterations